LiveLite/recommendation_tool/diet_recommendation/food.py

```diff
@@ -0,0 +1,236 @@
+# this module is created to define various methods related to diet and nutrition
+# below are functions to print recommended quantity of food groups to consume per day 
+# the functions will return none when input is not valid 
+import pandas as pd
+import numpy as np
+
+def recommendedProtein(age_in,gender_in):
+    if (age_in<9) & (age_in>100):
+        return None
+    if (gender_in != "m") & (gender_in != "f"):
+        return None
+    min = 0
+    max = 0
+
+    if gender_in =="f":
+        if 9 <= age_in <= 13:
+            min =4
+            max =6
+        if 14 <= age_in <= 30:
+            min =5
+            max =6.5
+        if age_in >= 31:
+            min =5
+            max =6
+
+    if gender_in =="m":
+        if 9 <= age_in <= 13:
+            min = 5
+            max = 6.5
+        if 13 < age_in <= 18:
+            min = 5.5
+            max = 7
+        if 18< age_in <= 30:
+            min = 6.5
+            max = 7
+        if 30 < age_in <=59:
+            min = 6
+            max= 7
+        if age_in > 59:
+            min = 5.5
+            max = 6.5
+
+    return(round(min*28.34), round(max*28.34), "grams")
+
+print(recommendedProtein(23,"m"))
+
+def recommendedGrains(age_in,gender_in):
+    if (age_in<9) & (age_in>100):
+        return None
+    if (gender_in != "m") & (gender_in != "f"):
+        return None
+    min_g = 0
+    min_wg =0 
+    max_g = 0
+    max_wg = 0
+
+    if gender_in =="f":
```

Comment:
Hopefully quick fix, could you encode gender to binary? If that's too much then don't worry about it.